From 08206c661b3a539809ddca26bc46cfb55568b3b3 Mon Sep 17 00:00:00 2001
From: Renato Botelho <garga@FreeBSD.org>
Date: Fri, 15 Apr 2016 11:38:50 -0300
Subject: [PATCH] Revert "Don't shift negative values as this is undefined."

It introduced a regression. Reverting fixes issue #1412

This reverts commit 5f152a041d02cbc454449889d4eb6c5a56bc031b.
---
 src/pkgcli.h | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/src/pkgcli.h b/src/pkgcli.h
index ec4a056..dbb7635 100644
--- src/pkgcli.h
+++ src/pkgcli.h
@@ -235,16 +235,16 @@ void usage_config(void);
 #define INFO_ALL	(((INFO_LASTFIELD) << 1) - 1)
 
 /* Identifying tags */
-#define INFO_TAG_NAME		(1LL<<32)
-#define INFO_TAG_ORIGIN		(1LL<<33)
-#define INFO_TAG_NAMEVER	(1LL<<34)
+#define INFO_TAG_NAME		(1LL<<60)
+#define INFO_TAG_ORIGIN		(1LL<<61)
+#define INFO_TAG_NAMEVER	(1LL<<62)
 
 /* Output YAML format */
-#define INFO_RAW		(1LL<<35)
-#define INFO_RAW_YAML		(1LL<<36)
-#define INFO_RAW_JSON		(1LL<<37)
-#define INFO_RAW_JSON_COMPACT	(1LL<<38)
-#define INFO_RAW_UCL		(1LL<<39)
+#define INFO_RAW		(-1LL<<63)
+#define INFO_RAW_YAML		(-1LL<<62)
+#define INFO_RAW_JSON		(-1LL<<61)
+#define INFO_RAW_JSON_COMPACT	(-1LL<<60)
+#define INFO_RAW_UCL		(-1LL<<59)
 
 /* Everything in the 'full' package output */
 #define INFO_FULL	(INFO_NAME|INFO_VERSION|INFO_INSTALLED|INFO_ORIGIN| \
